Dockerfile

```
@@ -16,7 +16,7 @@ RUN addgroup --gid 101 openvpn \
 
 COPY --chown=openvpn cert/pki/ca.crt /etc/openvpn/server/ca.crt
 COPY --chown=openvpn cert/pki/crl.pem /etc/openvpn/server/crl.pem
-COPY --chown=openvpn cert/ccd /etc/openvpn/server/ccd
+COPY --chown=openvpn cert/ccd /home/openvpn/available-ccd-folders
```

```suggestion
COPY --chown=openvpn cert/ccd /home/openvpn/available-ccds
```

Nit: I'm not sure how to properly explain it but I have a feeling that having the word `folder` inside a folder name is confusing. WDYT with the plural form?